spec/mailers/tigerdata_spec.rb

```diff
@@ -47,9 +47,11 @@
     mail = ActionMailer::Base.deliveries.last
 
     project.reload
-    expect(project.metadata).to include("submission")
-    expect(project.metadata["submission"]).to include("requested_by")
-    expect(project.metadata["submission"]).to include("request_date_time")
+    # These fields are not currently, nor ever were a part of the project metadata
```

Comment:
These really are part of the metadata standard see https://docs.google.com/document/d/1RDCowxE2ulfkh0GrqTln8Tr9t5XcWOZ2U--zRpiwe48/edit#heading=h.jnv7e4pqqw3u

Reply:
Good catch. I've restored them in the last commit https://github.com/pulibrary/tiger-data-app/pull/816/commits/44a62de63984ffb1eb2e46772427778c162879a2